File: Beecrowd/1180/1180.py
# Problem statement: https://www.beecrowd.com.br/judge/en/problems/view/1180
lowest = 1000
position = 0
N = int(input())
arr = list(map(int, input().split()))
for i in range(N):
    if arr[i] < lowest:
        lowest = arr[i]
        position = i
        
print(f'Menor valor: {lowest}\nPosicao: {position}')

File: Beecrowd/1180/1180.cpp
/* Problem statement: https://www.beecrowd.com.br/judge/en/problems/view/1180 */
#include <iostream>
using namespace std;
 
int main(){
	int N, num, lowest = 10000, pos;
	cin >> N;
	int arr[N];
	for(int i = 0; i < N; i++){
		cin >> num;
		if(num < lowest){
			lowest = num;
			pos = i;
		}
		arr[i] = num;
	}
	printf("Menor valor: %d\nPosicao: %d\n", lowest, pos);

	return 0;
}

